OZON.ru.js

```diff
@@ -1,563 +1,346 @@
-	for "Книги" and "Цифровые книги" ozon.ru site sections.
-
-	Change log
-	==========
-
-	2013-05-27
-
-	* New page layout parsing.
-
-	2012-08-11
-
-	* Corrected detail description parsing to accommodate the new book
-	  description pages template.
-	* Corrected "edition number" recognition.
-	* Introduced change log and to do comment sections.
-	* Few minor issue fixes.
-
-	2012-04-02
-
-	* Initial release.
-
-	TODO
-	====
-
-	* Develop more universal algorithm for "edition number" recognition.
-	  Currently it only recognises up to 10 editions in Russian verbal
-	  representation.
-
-	Legal notice
-	============
-
-	This program is free software: you can redistribute it and/or
-	modify it under the terms of the GNU Affero General Public License
-	as published by the Free Software Foundation, either version 3 of
-	the License, or (at your option) any later version.
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
 
 	This program is distributed in the hope that it will be useful,
 	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-	Affero General Public License for more details.
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
 
-	You should have received a copy of the GNU Affero General Public
-	License along with this program.  If not, see
-	<http://www.gnu.org/licenses/>.
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
 */
 
 function detectWeb(doc, url) {
-	var itemTypeXPath = '//ul[@class="navLine"]/li[1]/a';
-	var itemTypeDOMNode =
-	doc.evaluate(itemTypeXPath, doc, null,
-	XPathResult.ANY_TYPE, null).iterateNext();
-	var itemType;
-	if (itemTypeDOMNode) itemType = itemTypeDOMNode.textContent;
-	else return "";
-	if (itemType == "Книги") {
-		var nameXPath = '//div[@class="techDescription"]/div/div[2]/span';
-		var valueXPath = '//div[@class="techDescription"]/div/div[3]/span';
-		var nameXPathRes = doc.evaluate(
-		nameXPath, doc, null, XPathResult.ANY_TYPE, null);
-		var valueXPathRes = doc.evaluate(
-		valueXPath, doc, null, XPathResult.ANY_TYPE, null);
-		if (!IsAudioBook(nameXPathRes, valueXPathRes)) return "book";
-	} else {
-		itemTypeXPath = '//ul[@class="navLine"]/li[2]/a';
-		itemTypeDOMNode =
-		doc.evaluate(itemTypeXPath, doc, null,
-		XPathResult.ANY_TYPE, null).iterateNext();
-		if (itemTypeDOMNode) itemType = itemTypeDOMNode.textContent;
-		else return "";
-		if (itemType == "Цифровые книги") return "book";
-	}
-}
+	var hasIsbn = ZU.xpathText(doc, '//*[@itemprop="isbn"]') !== null;
+	var hasContent = ZU.xpath(doc, '//div[@class="bContentBlock"]') !== null;
```

Comment:
This is always true. `ZU.xpath` always returns an array, though it may be empty.